### PR TITLE
Add height to select

### PR DESCRIPTION
**Pull request by CodeBrauer**
**Before:**
bildschirmfoto 2016-06-24 um 12 20 42
**After**
bildschirmfoto 2016-06-24 um 12 20 37

### CODE REVIEW

File: src/forms.css [hunk 1]
```diff
@@ -17,6 +17,11 @@ input[type=text], select {
     font-family: 'Open Sans';
 }
 
+select {
+  height: 40px;
+  background: transparent;
```
Comment (CodeBrauer) [inline src/forms.css]: [This is special for macOS](http://stackoverflow.com/a/12501707/2767886)

**Comment by KingPixil** [comment]
Awesome job! I will merge this and run the build.
(You can do the build by running `npm install && gulp`)

It seems to work on Chrome the same, which is awesome!

Thanks for the bug fix, @CodeBrauer :smile:

**Comment by CodeBrauer** [comment]
No big deal, glad I helped.

Btw: I don't like to run the build in a PR so the diff stays clean and easier to read. But this is a personal preference.

**Comment by KingPixil** [comment]
Got it, thanks :+1:

**Pull request merged**